src/restconfig/src/main/java/org/geoserver/rest/catalog/LayerGroupController.java

```diff
@@ -198,6 +203,11 @@ public boolean supports(MethodParameter methodParameter, Type targetType, Class<
 
     @Override
     public void configurePersister(XStreamPersister persister, XStreamMessageConverter converter) {
+        // add keyword object configuration
+        XStream xstream = persister.getXStream();
```

Comment:
ServiceInfo and ResourceInfo will return something like this:

```
<keywords>
	<string>WFS</string>
	<string>WMS</string>
	<string>GEOSERVER</string>
</keywords>
```

but with the default configuration, layer goups controller return this instead:

```
<keywords>
	<org.geoserver.catalog.Keyword>keyword</org.geoserver.catalog.Keyword>
</keywords>
```
and could not understand why, anyway I agree with you all should use the same tag name.